tests-extension/test/e2e/testdata/workloads/customsub.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: subscription-template
+objects:
+- kind: Subscription
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
```

YAMLlint (1.38.0)

[error] duplication of key "kind" in mapping

(key-duplicates)